test/runtests.jl

```diff
@@ -1,2 +1,61 @@
 using GigaSOM
 using Test
+using Random
+# cd("..")
```

Comment:
this can also be safely removed.

Reply:
if I remove the `using Random` it says that Random is not defined when I tried to run it

Reply:
I was referring to `# cd("..")`

Reply:
ok, just because `using Random` was also selected